file_paths.md

```diff
@@ -59,6 +59,11 @@ The files annotated by an '*' indicate that they are created by package installa
         routes.yaml                       # :route_file
         ssl                               # :ssldir                      /etc/puppet/ssl
 
+    /etc/puppetlabs/pxp-agent *
+        modules.d *                       # directory where pxp-agent modules are stored
+            puppet.cfg *                  # /opt/puppetlabs/pxp-agent/modules/puppet configuration
+        pxp-agent.cfg                     # pxp-agent configuration file
```

Comment:
I suggest we use the extension `.conf` going forward rather than `.cfg`.

The former is consistent with:
* puppet
* puppetdb
* puppetserver
* misc new PE tools coming out soon like `puppet-job` and `puppet-access`

Thoughts?